README.md

```diff
@@ -270,6 +287,85 @@ rather than having it statically provided in their respective config file, then
 `chariott_uri: "http://0.0.0.0:50000"`<br>
 1. In the consumer's config file and the provider's config file, remove the setting for invehicle_digital_twin_uri, so that the chariott_uri will be used to find the In-vehicle Digital Twin URI.<br>
 
+## <a name="running-in-a-container">Running in a Container</a>
+
+Below are the steps for running the service in a container. Note that the configuration files used
+by the containerized service are cloned from [container/config](./container/config/) defined in the
+project's root.
+
+### <a name="docker">Docker</a>
+
+<b>Prerequisites</b>
+
+[Install Docker](https://docs.docker.com/engine/install/)
+
+<b>Running in Docker</b>
+
+To run the service in a Docker container:
+
+1. Run the following command in the project's root directory to build the docker container from the
+Dockerfile:
+
+    ```shell
+    docker build -t invehicle_digital_twin -f Dockerfile .
+    ```
+
+1. Once the container has been built, start the container in interactive mode with the following
+command in the project's root directory:
+
+    ```shell
+    docker run --name invehicle_digital_twin -p 5010:5010 --env-file=./container/config/docker.env --add-host=host.docker.internal:host-gateway -it --rm invehicle_digital_twin
+    ```
+
+1. To detach from the container, enter:
+
+    <kbd>Ctrl</kbd> + <kbd>p</kbd>, <kbd>Ctrl</kbd> + <kbd>q</kbd>
+
+1. To stop the container, enter:
+
+    ```shell
+    docker stop invehicle_digital_twin
+    ```
+
+### <a name="podman">Podman</a>
+
+<b>Prerequisites</b>
+
+[Install Podman](https://podman.io/docs/installation)
+
+<b>Running in Podman</b>
+
+To run the service in a Podman container:
+
+1. Run the following command in the project's root directory to build the podman container from the
+Dockerfile:
+
+    ```shell
+    podman build -t invehicle_digital_twin:latest -f Dockerfile .
+    ```
+
+1. Once the container has been built, start the container with the following command in the
+project's root directory:
+
+    ```shell
+    podman run -p 5010:5010 --env-file=./container/config/podman.env --network=slirp4netns:allow_host_loopback=true localhost/invehicle_digital_twin
+    ```
+
+1. To stop the container, run:
+
+    ```shell
+    podman ps -f ancestor=localhost/invehicle_digital_twin:latest --format="{{.Names}}" | xargs podman stop
+    ```
+
+### <a name="dockerfile">Dockerfile</a>
```

Comment:
Maybe move this `Dockerfile` section just after the paragraph in the `Running in a Container` section since the paragraph under the `Running in a container` section and the `Dockerfile` section discusses what's contained in the root directory.

Also I think it might be clearer to the user as to why there are two Docker files before they proceed to the `Docker` and `Podman` sections to run the service as a container.

Reply:
Sure, I can do that

Reply:
updated